ontology/yaml/resources/METERS/entity_types/FM.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FM_STANDARD:
+  description: "Standard flow meter typically for liquid."
+  implements:
+  - FM
+  is_canonical: true
+  uses:
+  - volume_accumulator
+  opt_uses:
+  - failed_alarm
+  - flowrate_sensor
```

Comment:
Is this really optional? This is a flow meter, wouldnt flowrate be a required field for standard flow meters?

Think its better to have this as required, and to use MISSING translation fields if necessary for non-compliant entities.

Reply:
Done with required changes